src/main/java/uk/gov/companieshouse/confirmationstatementapi/controller/StatementOfCapitalController.java

```diff
@@ -25,13 +27,19 @@ public StatementOfCapitalController(StatementOfCapitalService statementOfCapital
     }
 
     @GetMapping("/confirmation-statement/company/{companyNumber}/statement-of-capital")
-    public ResponseEntity<StatementOfCapitalJson> getStatementOfCapital(@PathVariable String companyNumber) {
+    public ResponseEntity<StatementOfCapitalJson> getStatementOfCapital(
+            @PathVariable String companyNumber,
+            @RequestHeader(value = ERIC_REQUEST_ID_KEY) String requestId) {
+
+        var logMap = new HashMap<String, Object>();
+        logMap.put("company_number", companyNumber);
+
         try {
-            LOGGER.info("Calling service to retrieve statement of capital data");
+            LOGGER.infoContext(requestId, "Calling service to retrieve statement of capital data.", logMap);
             var statementOfCapital = statementOfCapitalService.getStatementOfCapital(companyNumber);
             return ResponseEntity.status(HttpStatus.OK).body(statementOfCapital);
         } catch (ServiceException e) {
-            LOGGER.error("Error retrieving statement of capital data ", e);
+            LOGGER.errorContext(requestId, "Error retrieving statement of capital data.", e, logMap);
```

Comment:
This block is capturing an exception but returning a 404 not found - is this an exceptional case (should be a 500 response code) or a client error (change log level to info)?

Reply:
Created a separate bug ticket ([NCS-777](https://companieshouse.atlassian.net/browse/NCS-777)) to address this.